LICENSE.md

```diff
@@ -1,3 +1,6 @@
+The MIT License (MIT)
```

Comment:
Why changin header ?

Reply:
I just added the title like in our other MIT licenced projects